components/Head/Head.tsx

```diff
@@ -38,6 +38,7 @@ export interface HeadProps {
   titleSuffix?: string;
   url?: string;
   noIndex?: boolean;
+  keywords?: Array<String>;
```

Comment:
I prefer the syntax 
```suggestion
  keywords?: string[];
```

Reply:
Fine by me. I don't care either way, just went with what the editor said would fix my syntax.